docs.yaml

```diff
@@ -32,6 +32,8 @@ sidebar:
       - "/cli/commands/unlink"
     - - Login
       - "/cli/commands/login"
+    - - Login
+      - "/cli/commands/token"
```

Comment:
Why do we have this duplicated? @mhadaily

Reply:
ah that supposed to be Token :D